File: Baekjoon/solved_ac/class2/4153.py
# 4153: 직각삼각형

while (True):
  a, b, c = list(map(int, input().split()))

  if (a*b*c == 0): break

  if (a*a + b*b == c*c or
      a*a + c*c == b*b or
      b*b + c*c == a*a): print("right")
  else: print("wrong")

File: Baekjoon/solved_ac/class2/4153.cpp
// 4153: 직각삼각형

#include <iostream>
using namespace std;

int main() {
  
  while (true) {
    int a, b, c;
    cin >> a >> b >> c;

    if (a * b * c == 0) break;

    if (a*a + b*b == c*c ||
        a*a + c*c == b*b ||
        b*b + c*c == a*a) {
      cout << "right" << endl;
    } else {
      cout << "wrong" << endl;
    }
  }

  return 0;
}